testdata/test-data.xml

```diff
@@ -58,7 +58,7 @@
             <boolean permissions="prop-default">true</boolean>
         </boolean-prop>
         <date-prop name=":hasDate">
-            <date permissions="prop-default" >JULIAN:CE:1401-05-17:CE:1402-01</date>
+            <date permissions="prop-default" >JULIAN:BCE:0700:BCE:0600</date>
```

Comment:
in the failed test it complains: 

> Message:{"knora-api:error":"org.knora.webapi.exceptions.BadRequestException: Start date 0600 BCE is after end date 0700 BCE","@context":{"knora-api":"http://api.knora.org/ontology/knora-api/v2#"}}!

https://github.com/dasch-swiss/dsp-tools/runs/2129275872#step:8:1448